### ISSUE TITLE
Previewing is broken

### ISSUE THREAD

**Issue by benjyw** [pantsbuild/pantsbuild.org#152]
Following the instructions in the readme for building the site and previewing it in a dev server, on current main, yields this error:

```
ERROR
Cannot read properties of undefined (reading 'label')
TypeError: Cannot read properties of undefined (reading 'label')
    at DocsVersionDropdownNavbarItem (webpack-internal:///./src/theme/NavbarItem/DocsVersionDropdownNavbarItem.js:132:36)
    at renderWithHooks (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:16305:18)
    at mountIndeterminateComponent (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:20069:13)
    at beginWork (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:21582:16)
    at HTMLUnknownElement.callCallback (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:4164:14)
    at Object.invokeGuardedCallbackDev (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:4213:16)
    at invokeGuardedCallback (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:4277:31)
    at beginWork$1 (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:27446:7)
    at performUnitOfWork (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:26555:12)
    at workLoopConcurrent (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:26538:5)
```

<img width="1289" alt="Screenshot 2024-02-15 at 8 02 13 PM" src="https://github.com/pantsbuild/pantsbuild.org/assets/512764/37eb31af-efb1-4cd0-a142-3cf51295b54a">

**Comment by huonw** [comment]
Was this fixed by https://github.com/pantsbuild/pantsbuild.org/pull/155 ?

**Comment by benjyw** [comment]
Ah yes! Fixed by #155
